shell/platform/common/public/flutter_texture_registrar.h

```diff
@@ -34,6 +34,10 @@ typedef struct {
   size_t width;
   // Height of the pixel buffer.
   size_t height;
+  // An optional callback that gets invoked when the |buffer| can be released.
+  void (*release_callback)(void* release_context);
+  // Opaque data passed to |release_callback|.
+  void* release_context;
```

Comment:
Adding these caused undefined behavior for existing code (unless the code was specifically written to defend against the possibility of new values by zero-initializing the entire memory block).

If we edit this again we should add a struct size to do safe reads, as we do with embedder.h.